assets/src/edit-story/components/library/panes/pageTemplates/templateSave.js

```diff
@@ -76,9 +67,9 @@ const SaveButton = styled.button`
 `;
 
 const IconWrapper = styled.div`
-  width: 80px;
-  height: 106px;
-  margin-top: 34px;
+  width: 32px;
+  height: 42px;
+  margin-right: 29px;
   svg {
     color: ${({ theme }) => theme.colors.fg.tertiary};
 
```

Comment:
📣   _Save yourself from many lines of annoying CSS by [this one weird trick](https://github.com/google/web-stories-wp/blob/main/.storybook/main.cjs#L77), that doctors hate!_ 📣

(put the SVG in a subfolder named `images/` and colors defined in the SVG will be preserved - for exactly this type of purpose!)

((I do recognize that this hardcodes the colors in the SVG and doesn't make it easily skinnable if we ever were to do a light-mode version, but that's a minor worry - and probably a worry for many other graphics as well))

Reply:
Thanks!